shared/config/config.go

```diff
@@ -0,0 +1,111 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const DefaultGenesis = "build/config/genesis.json"
```

Comment:
Is this planning on being used? If not - let's remove

Reply:
tl;dr removed the constant but keeping the file for now.

It's still being used in the config files and needed for the consensus prototype, but the constant was not being used.